src/Elasticsearch.Net/Transport/Transport.cs

```diff
@@ -33,8 +33,7 @@ public Transport(TConnectionSettings configurationValues)
 			TConnectionSettings configurationValues,
 			IRequestPipelineFactory pipelineProvider,
 			IDateTimeProvider dateTimeProvider,
-			IMemoryStreamFactory memoryStreamFactory
-			)
+			IMemoryStreamFactory memoryStreamFactory)

```

Comment:
We either single line or explode all e.g

```csharp
method(a, b, c , d, e);
```

or 

```
method(
    a,
    b, 
    c,
    d
);
```

no cutoffs

```
method(a,b,c
       d);
```

as also occuring in other places here.